docs/sources/flow/reference/components/loki.source.file.md

```diff
@@ -104,26 +110,30 @@ configuration.
 ## Debug information
 
 `loki.source.file` exposes some target-level debug information per reader:
-* The tailed path.
-* Whether the reader is currently running.
-* What is the last recorded read offset in the positions file.
+
+- The tailed path.
+- Whether the reader is currently running.
+- What is the last recorded read offset in the positions file.
 
 ## Debug metrics
-* `loki_source_file_read_bytes_total` (gauge): Number of bytes read.
-* `loki_source_file_file_bytes_total` (gauge): Number of bytes total.
-* `loki_source_file_read_lines_total` (counter): Number of lines read.
-* `loki_source_file_encoding_failures_total` (counter): Number of encoding failures.
-* `loki_source_file_files_active_total` (gauge): Number of active files.
+
+- `loki_source_file_read_bytes_total` (gauge): Number of bytes read.
+- `loki_source_file_file_bytes_total` (gauge): Number of bytes total.
+- `loki_source_file_read_lines_total` (counter): Number of lines read.
+- `loki_source_file_encoding_failures_total` (counter): Number of encoding failures.
+- `loki_source_file_files_active_total` (gauge): Number of active files.
 
 ## Component behavior
```

Comment:
I think that this section should be amended to mention the option that you can tail?